pkg/store/instance.go

```diff
@@ -52,6 +57,9 @@ type Instance struct {
 	DriverPID       int                `json:"driverPID,omitempty"`
 	Errors          []error            `json:"errors,omitempty"`
 	Config          *limayaml.LimaYAML `json:"config,omitempty"`
+	RootFsPath      string             `json:"rootfs,omitempty"`
+	DistroName      string             `json:"distroName,omitempty"`
```

Comment:
Why do we need these values here?

Reply:
We don't, removed in latest revision. It's leftover from prototyping